modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -56,58 +56,60 @@ Below you can see what the core runtime’s hierarchy looks like:
 
 image::error-hierarchy.png[Error Hierarchy]
 
-All errors are either general or `CRITICAL`, the latter being so severe that they cannot be handled and will only be logged. At the top of the general hierarchy is `ANY`, which allows for matching all types under it. It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow for specification of unclear errors in the future, without changing the existing behavior of the app.
+All errors are either general or `CRITICAL`, the latter being so severe that they cannot be handled and will only be logged. At the top of the general hierarchy is `ANY`, which allows for matching all types under it. It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow for the specification of unclear errors in the future, without changing the existing behavior of the app.
```

Comment:
> It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow for the specification of unclear errors in the future, without changing the existing behavior of the app.

This is probably too verbose. Something more specific about the importance of the unknown type would be easier to read :

> Whenever there is no clear reason for the failure, a component can throw a `UNKOWN` type. This type can only be handled by an `ANY` type so that you can specify unclear errors in the future without changing the already existing behavior of your application.

Moreover:
> so that you can specify unclear errors in the future without changing the already existing behavior of your application.
I can't find a better way of rephrasing this. Somehow the purpose of catching these errors using the ANY type gets lost in the vagueness of this sentence.